src/components/Approval/Approval.js

```diff
@@ -64,64 +62,38 @@ const Approval = ({
 
   return (
     <>
-      {assetNumber === '1' && (
-        <>
-          {BN(web3.allowance1.toString()).comparedTo(txnAmount) === -1 && (
-            <>
-              <Button
-                className="btn-fill w-100 h-100"
-                color="neutral"
-                onClick={() => {
-                  handleApproval()
-                }}
-              >
-                <i className="icon-extra-small icon-lock icon-light align-middle" />
-                <br />
-                Approve {symbol}
-              </Button>
-            </>
-          )}
-          {BN(web3.allowance1.toString()).comparedTo(txnAmount) === 1 && (
-            <>
-              <Button className="btn-fill w-100 h-100" color="success">
-                <i className="icon-extra-small icon-lock icon-dark align-middle" />
-                <i className="icon-extra-small icon-check icon-light align-middle" />
-                <br />
-                {symbol} {t('ready')}
-              </Button>
-            </>
-          )}
-        </>
-      )}
-      {assetNumber === '2' && (
-        <>
-          {BN(web3.allowance2.toString()).comparedTo(txnAmount) === -1 && (
-            <>
-              <Button
-                className="btn-fill w-100 h-100"
-                color="neutral"
-                onClick={async () => {
-                  handleApproval()
-                }}
-              >
-                <i className="icon-extra-small icon-lock icon-light align-middle" />
-                <br />
-                Approve {symbol}
-              </Button>
-            </>
-          )}
-          {BN(web3.allowance2.toString()).comparedTo(txnAmount) === 1 && (
-            <>
-              <Button className="btn-fill w-100 h-100" color="success">
-                <i className="icon-extra-small icon-lock icon-dark align-middle" />
-                <i className="icon-extra-small icon-check icon-light align-middle" />
-                <br />
-                {symbol} ${t('ready')}
-              </Button>
-            </>
-          )}
-        </>
-      )}
+      {assetNumber === '1' &&
+        BN(web3.allowance1.toString()).comparedTo(txnAmount) === -1 && (
+          <Col>
```

Comment:
Move the <Col> inside the Approval's check to ensure it doesn't show an empty <Col> inside the areas where this component is injected (remove the <Col>s from the views now